uber/models.py

```diff
@@ -379,6 +379,82 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
             if not ignore_csrf:
                 check_csrf(params.get('csrf_token'))
 
+    def timespan(self, minute_increment=60):
```

Comment:
When possible, I'd like to keep HMTL code out of ``models.py``, for two main reasons:

1) The model-related code here is already huge and represents a significant fraction of the code in this codebase.  Moreover, it's the file that people tend to most commonly have open and check a lot.  So keeping it as tight as possible is good, i.e. having only model-related code here rather than presentation code.

2) It makes it harder for people to know where to look to find stuff.  Right now, the answer to "where do I go to find database-related stuff" is "models.py" and "where do I go to find Python code that spits out HTML" is "custom_tags.py" or whatever that's called now, etc.  Our codebase isn't that huge, but it's still big enough that we should work hard to keep things well-organized, and this kind of mixing makes me nervous.

I wouldn't necessarily withhold a +1 from this PR over this issue, since it's only a couple of stray methods that can probably be cleaned up later.  but it definitely makes me uncomfortable and I'd prefer to see these as custom tags/methods/whatever alongside the rest of our Jinja code.

(I suppose that this particular ``timespan`` method is probably fine here in ``models.py`` since it's not actually HTML-specific, but what I'm saying definitely applies to the next couple of methods.)